### DISCUSSION TITLE
Remove tag opens dropdown blocking buttons

### DISCUSSION THREAD

**Discussion by smseidl** [paperless-ngx/paperless-ngx#2465] [Feature Requests] [answered]
So, I ran across a little annoyance tonight and thought maybe someone would have an idea for a fix.  As I'm going thru my documents and removing the inbox tag, the tags drop down pops up every single time.  It would be wonderful if:
1. Tags could be removed from a document without the dropdown opening
2. The dropdown would not cover the save/save next buttons.

![image](https://user-images.githubusercontent.com/23556348/213354976-5fb1e252-cec5-483a-8302-cb4396afeadb.png)

I know it's a minor thing, but it does cost time to have to click outside of the dropdown for it to close and then click on Save/Next.

**Comment by ronaldheft** [comment]
+1 to this. Opening the dropdown makes sense when you start typing, but not when you click the "x" to remove a tag.

**Comment by hendrik1120** [comment]
I might be missing something here, but why isn't selecting all the documents and then mass editing the tags an option for you?

But agreed, it's indeed pretty annoying.

**Reply by smseidl** [reply]
Most of my documents are related to medical bills/eobs/etc.  I have 2 other systems that I track these in (one older windows version that is lacking updates, and a version I'm writing in Django).  I go thru the documents 1 at a time to add them into these apps and then remove the 'inbox' tag when done processing.  So using the mass editing won't work with my other processes.

**Comment by shamoon** [comment, marked as answer]
Please see https://github.com/paperless-ngx/paperless-ngx/pull/2625 =)
